chrome/validate-nav.js

```diff
@@ -6,16 +6,16 @@ const navigationSchema = require('./validationSchemas/navigation');
 
 
 
-const navigationFiles = ['ansible', 'application-services', 'rhel', 'settings', 'user-preferences', 'openshift'].map((file) => path.resolve(__dirname, `${file}-navigation.json`))
+const navigationFiles = fs.readdirSync(__dirname)
+  .filter(f => f.endsWith('-navigation.json'))
+  .filter(f => f !== 'landing-navigation.json');
```

Comment:
Two extra iterations over the same array. I'd merge it into one `.filter(f => f !== 'landing-navigation.json' && f.endsWith('-navigation.json'))`

Reply:
Done.